packages/workspace/src/core/affected-project-graph/locators/workspace-projects.ts

```diff
@@ -50,5 +51,5 @@ export const getImplicitlyTouchedProjects: TouchedProjectLocator = (
     }
   }
 
-  return touched;
+  return [...new Set(touched)];
```

Comment:
Can we replace `touched = []` with `touched = new Set()` and return `Array.from(touched)`? It avoids having to convert twice.